library/src/engine.cxx

```diff
@@ -265,32 +265,40 @@ std::map<std::string, bool> engine::getRenderingBackendList()
 //----------------------------------------------------------------------------
 void engine::loadPlugin(const std::string& pathOrName, const std::vector<fs::path>& searchPaths)
 {
-  if (pathOrName.empty())
+  std::string localPathOrName = pathOrName;
+
+  if (localPathOrName.empty())
   {
     return;
   }
 
+  // For easier removal when removing deprecation: F3D_DEPRECATED
+  if (localPathOrName == "exodus") {
+    f3d::log::warn("The 'exodus' plugin is deprecated, loading 'hdf' instead");
+    localPathOrName = "hdf";
```

Comment:
Sorry about that, recursive call was fine after all, I misunderstood what you implemented initially.

Reply:
reverted last change